openstack-k8s-operators/placement-operator
# Clean up CI jobs

* unify make test and gotest targets
* remove redundant github actions preferring their prow counterparts
* ensure image build can be fixed if fails 
* add a small check that prevents merging go.mod replace lines
